### DISCUSSION TITLE
the hawku plugin which is noise reduction not working ?

### DISCUSSION THREAD

**Discussion by Mingu201** [OpenTabletDriver/OpenTabletDriver#1285] [Q&A] [answered]
so i installed hawku plugin and when i turn on the noise reduction , the driver suddenly error

**Comment by jamesbt365** [comment, marked as answer]
I cannot reproduce this, this may of just been a thing in the past but now i cannot get it to do it, if you are on windows you probably messed with the timer system by debloating your os or other things, but this doesn't occur now even on a windows virtual machine or my linux machine.
